services/common/src/components/documents/DocumentTable.tsx

```diff
@@ -368,6 +368,7 @@ export const DocumentTable: FC<DocumentTableProps> = ({
     : {};
 
   const coreTableProps = {
+    rowKey: "document_manager_guid",
```

Comment:
I got some errors about keys when I uploaded documents so I threw this in. Judging by the updated snap tests, some components were using mine_document_guid.